runtime/doc/options.txt

```diff
@@ -6033,6 +6033,52 @@ A jump table for the options with a short description can be found at |Q_op|.
 	In case of buffer changing commands the cursor is placed at the column
 	where it was the last time the buffer was edited.
 
+				*'statuscolumn'* *'stc'*
+'statuscolumn' 'stc'	string	(default: empty)
+			local to window
+	EXPERIMENTAL
+	When this option is not empty, it determines the content of the area to
```

Comment:
```suggestion
	When non-empty, determines the content of the area to
```

Reply:
Sure, the same wording is used for `iconstring`, `indentexpr`, `rulerformat` and `titlestring` btw, that's why I used it. `statusline` on the other hand reads:

    	When non-empty, this option determines the content of the status line.

I guess I'll word it similar to that.

Reply:
:+1: